docs/user-guide/reading-and-writing-files.ipynb

```diff
@@ -61,6 +61,118 @@
     "b"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## CSV\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "CSV support requires [pandas](<https://pandas.pydata.org/>) which must be installed separately.\n",
```

Comment:
Should this be
```suggestion
    "CSV support requires [pandas](https://pandas.pydata.org/) which must be installed separately.\n",
```
?

Reply:
Yes. Strange that linkcheck didn't find this.